.eslintignore

```diff
@@ -1,2 +1,4 @@
-# Generated file:
-/parser.js
+node_modules
+dist
+
+!.eslintrc.js
```

Comment:
I don't see why this needs to be explicitly included. It's not otherwise excluded.

Reply:
Hidden files are disabled from linting by default, so when including an ignore file, one needs to opt back in.

Reply:
Oh thanks, I didn't know that.